### ISSUE TITLE
night sunny

### ISSUE THREAD

**Issue by herauthon** [chubin/wttr.in#909]
Sometimes there is a combination of night (table time) and sunny (zonnig)
which is not really correct - since it cannot be sunny at night time..
but maybe this should be "clear" - devoid of clouds, mist or anything else..
a clear night is possible..
so

[ "${table_3}"  ] && [ "${status}" == "sunny" ] && export status="clear"  
[ "${table_4}"  ] && [ "${status}" == "sunny" ] && export status="clear"  

correct ?

**Comment by chubin** [comment]
Agree. It is a recurring issue, and we should find a good solution here.

Duplicates:

* #1089

**Comment by fredmo** [comment]
still present
